pkg/notification/notification_builder_test.go

```diff
@@ -0,0 +1,142 @@
+package notification
+
+import (
+	"testing"
+
+	"github.com/gofrs/uuid"
+
+	"github.com/codeready-toolchain/api/api/v1alpha1"
+	test2 "github.com/codeready-toolchain/toolchain-common/pkg/test/usersignup"
```

Comment:
Probably better to use an alias that's related to usersignup for this. eg. `testusersignup`